platform/net/ip/interface_address_test.go

```diff
@@ -90,24 +91,44 @@ var _ = Describe("resolvingInterfaceAddress", func() {
 
 		Context("when IP was already resolved", func() {
 			BeforeEach(func() {
-				ipResolver.GetPrimaryIPv4IPNet = &gonet.IPNet{
+				ipResolver.GetPrimaryIPNet = &gonet.IPNet{
 					IP:   gonet.ParseIP("127.0.0.1"),
 					Mask: gonet.CIDRMask(16, 32),
 				}
 
-				_, err := interfaceAddress.GetIP()
+				_, err := interfaceAddress.GetIP(false)
 				Expect(err).ToNot(HaveOccurred())
 			})
 
 			It("does not attempt to resolve IP again", func() {
-				ipResolver.GetPrimaryIPv4InterfaceName = ""
+				ipResolver.GetPrimaryIPInterfaceName = ""
 
-				ip, err := interfaceAddress.GetIP()
+				ip, err := interfaceAddress.GetIP(false)
 				Expect(err).ToNot(HaveOccurred())
 				Expect(ip).To(Equal("127.0.0.1"))
 
-				Expect(ipResolver.GetPrimaryIPv4InterfaceName).To(Equal(""))
+				Expect(ipResolver.GetPrimaryIPInterfaceName).To(Equal(""))
 			})
 		})
+
+		Context("when GetIP was called with true or false", func() {
+
+			BeforeEach(func() {
+				ipResolver.GetPrimaryIPNet = &gonet.IPNet{
+					IP:   gonet.ParseIP("127.0.0.1"),
+					Mask: gonet.CIDRMask(16, 32),
+				}
+			})
+
+			for _, value := range []bool{true, false} {
```

Comment:
Can `value` be renamed to something that expresses intent (like whatever is chosen as the name of the `bool` where `GetIP()` is defined?